docs/configuration_requirements.rst

```diff
@@ -0,0 +1,19 @@
+Configuration requirements
+**************************
+
+.. only:: iso
+
+    This page shows the configuration requirements for the Inmanta Service Orchestrator |version_major| version.
+    These constraints are defined for the whole lifetime of this major version i.e.
+    they are not pinned to a specific three-digit version.
+
+
+    .. datatemplate:json:: required_configuration.json
```

Comment:
What if this file does not exist? We probably need an empty one in this repo so we can still build the docs stand-alone?